stub.py

```diff
@@ -0,0 +1,94 @@
+"""Stub class"""
```

Comment:
We can make this module docstring more descriptive maybe 
```
"""Defines interface that can be subclassed to use with the tidy3d webapi."""
```

Reply:
modified